### DISCUSSION TITLE
Struggling to install after v3?

### DISCUSSION THREAD

**Discussion by Samdoescode** [lukas-reineke/indent-blankline.nvim#692] [Q&A] [answered]
Hey there, 

I'm sure I'm doing something incredibly dumb here but I am unable to get indent line to work in v3 at all. I am currently calling using the recommend lines in the lazy block. 
`  {
    "lukas-reineke/indent-blankline.nvim",
    config = function()
      require("ibl").setup {}
    end,
  },`
Have also tried with 
`{ "lukas-reineke/indent-blankline.nvim", main = "ibl", opts = {} }`
and then 
`require("ibl").setup{}`
after the lazy block
both of which throw this error on start up
```
Failed to run `config` for indent-blankline.nvim

C:\Users\Sam\AppData\Local\nvim\init.lua:81: module 'ibl' not found:
```
I have been able to (I assume) use v2 by changing the install step to
`{ "lukas-reineke/indent-blankline.nvim", opts = {} }`
and the setup to
`require("indent_blankline").setup{}` (stolen from [Lukas Reineke](https://github.com/lukas-reineke) dot files)

Having exhausted my knowledge of lua and nvim (I'm very new to this), it seems like something I am doing is cursing an issue with the new entry point. 

As mentioned everything is working as expected with those changes but am looking at v2.20.8 and would love to have v3!

Please let me know if there is something obviously wrong here! Your help is much appreciated.

**Comment by lukas-reineke** [comment, marked as answer]
Sounds like you still have version 2 installed. Make sure you actually update. How to do that depends on the plugin manager you use

**Reply by Dananjay996** [reply]
Hey!, I am using lazy as the plugin manager. For more context, I am using kickstart nvim as the template. How do I install V3 and resolve the error?

**Comment by pdunc** [comment]
I had the same issue, the `lazy-lock.json` was pointing to an older commit.

To fix it:

1. Change the `indent-blankline.nvim` line in `lazy-lock.json` to `"indent-blankline.nvim": { "branch": "master", "commit": "29be0919b91fb59eca9e90690d76014233392bef" },`
2. Run `:Lazy`, press `U` to update

The error disappears 😄

**Reply by mgierada** [reply]
That worked for me!

**Reply by truongnm97** [reply]
This doesn't work for me, everytime I run `:Lazy` > `U`, the `lazy-lock.json`'s auto updated back to the old commit hash. But I'm able to change the commit hash by doing this in lazy config:
```lua
{
    "lukas-reineke/indent-blankline.nvim",
    main = "ibl",
    commit = "29be0919b91fb59eca9e90690d76014233392bef",
}
```

**Reply by Ashmeet7** [reply]
where to add this i am using lazyvim

**Reply by ManJack1** [reply]
-- Create a file named indent-blankline.lua in the lua/plugins folder
-- The file content is as follows:
return {
"lukas-reineke/indent-blankline.nvim",
main = "ibl",
commit = "29be0919b91fb59eca9e90690d76014233392bef",
}

**Reply by joshuablais** [reply]
This worked for me, thank you.
